debian/rules

```diff
@@ -146,6 +146,13 @@ endif
 	# Install libmariadbclient.a compatibility link
 	ln -s libmariadb.a $(TMP)/usr/lib/$(DEB_HOST_MULTIARCH)/libmariadbclient.a
 
+	# Symlink plugins that are used by both server and client and thus need to
+	# load from the libmariadb path as well
+	ln -s ../../../mysql/plugin/auth_test_plugin.so $(TMP)/usr/lib/$(DEB_HOST_MULTIARCH)/libmariadb3/plugin/auth_test_plugin.so
+	ln -s ../../../mysql/plugin/qa_auth_interface.so $(TMP)/usr/lib/$(DEB_HOST_MULTIARCH)/libmariadb3/plugin/qa_auth_interface.so
+	# Move test plugins that are only needed by the client to the libmariadb path
```

Comment:
fixed qa_auth_client.so with #1497. Why not just use that?

Reply:
So that PR is moving one plugin? We need one move and two symlinks, could all 3 be handled in CMake so we get rid of this debian/rules hacks? That would be optimal.

That is however not necessary right now. Merging in this PR should not be delayed.

Those CMake changes can be applied later and anyway such a PR needs to be based on this change and prove in extensive testing and our CI systems that the CMake changes were correct and did not regress anything.

Reply:
let's use #1497 if it works